cli/src/pixl_cli/_io.py

```diff
@@ -88,6 +88,11 @@ def _load_csv(filepath: Path) -> pd.DataFrame:
     messages_df = _map_columns(messages_df, MAP_CSV_TO_MESSAGE_KEYS)
     _raise_if_column_names_not_found(messages_df, [col.name for col in DF_COLUMNS])
 
+    if "participant_id" in messages_df:  # noqa: SIM401
+        messages_df["pseudo_patient_id"] = messages_df["participant_id"]
+    else:
+        messages_df["pseudo_patient_id"] = None
```

Comment:
why don't we require all inputs to have a `participant_id` and update the test input data so we don't need this branch of logic.